build.gradle

```diff
@@ -285,6 +404,21 @@ task integTestRemote(type: RestIntegTestTask) {
             includeTestsMatching "org.opensearch.flowframework.rest.*IT"
         }
     }
+
+    // Exclude integration tests that require security plugin
+    if (System.getProperty("https") == null || System.getProperty("https") == "false") {
+        filter {
+            excludeTestsMatching "org.opensearch.flowframework.rest.FlowFrameworkSecureRestApiIT"
+        }
+    }
+
+    // Include only secure integration tests in security enabled clusters
+    if (System.getProperty("https") != null && System.getProperty("https") == "true") {
+        filter {
+            includeTestsMatching "org.opensearch.flowframework.rest.FlowFrameworkSecureRestApiIT"
+            excludeTestsMatching "org.opensearch.flowframework.rest.FlowFrameworkRestApiIT"
+        }
+    }
```

Comment:
Why do we need to have these checks twice as already added on L309?

Reply:
This is to configure the remote integration test task, the previous checks are for local integration tests